P0009/P0009.md

```diff
@@ -1303,15 +1303,16 @@ namespace std {
 <b>22.7.� Overview [mdspan.terms]</b>
 
 [1]{.pnum} A _multidimensional index space_ is a Cartesian product of integer intervals.
-Each interval can be represented by a half-open range $[I_b, I_e)$, where $I_b$ and $I_e$ are the lower and upper bounds of the $i^{th}$ dimension.
+Each interval can be represented by a half-open range $[L_i, U_i)$, where $L_i$ and $U_i$ are the lower and upper bounds of the $i^{th}$ dimension.
 The _rank_ of a multidimensional index space is the number of intervals it represents.
-The _size_ of a multidimensional index space is the product of $I_e-I_b$ for each dimension if its rank is greater than $0$, and $1$ otherwise.
+The _size_ of a multidimensional index space is the product of $U_i-L_i$ for each dimension $i$ if its rank is greater than $0$, and $1$ otherwise.
+In the following representations of a multidimensional index space, such as `extents`, will be used synonymously with the index space itself.
```

Comment:
```suggestion
In what follows, any of various representations of a multidimensional index space, such as `extents`, will be used synonymously with the index space itself.
```

Reply:
I moved that sentence to extents intro itself.